From 12d44a55fe061ca06232007d1ceadf7f08ccedda Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Sat, 21 Feb 2026 19:18:14 +0100
Subject: [PATCH] add `miyoo` format compiler

---
 libretro-config.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libretro-config.sh b/libretro-config.sh
index 3a048e53..4ac783a4 100644
--- a/libretro-config.sh
+++ b/libretro-config.sh
@@ -984,6 +984,11 @@ case "$platform" in
 					FORMAT_COMPILER_TARGET="theos_ios"
 					DIST_DIR="theos_ios"
 					;;
+				miyoo)
+						FORMAT_EXT="so"
+						FORMAT_COMPILER_TARGET="miyoo"
+						DIST_DIR="unix"
+						;;
 				android)
 					FORMAT_EXT="so"
 					FORMAT_COMPILER_TARGET="android"
-- 
2.45.2.windows.1

