From 98427291024b36d8ae67bd1d57a2acd9b3fe558d Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Sat, 21 Feb 2026 19:20:20 +0100
Subject: [PATCH] fix tic-80 core rules

---
 rules.d/core-rules.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rules.d/core-rules.sh b/rules.d/core-rules.sh
index 9a157fe8..c19f6fcd 100644
--- a/rules.d/core-rules.sh
+++ b/rules.d/core-rules.sh
@@ -1504,7 +1504,7 @@ libretro_tic80_git_submodules="yes"
 libretro_tic80_build_subdir="core"
 libretro_tic80_build_extradir="lib/"
 libretro_tic80_build_rule="cmake"
-libretro_tic80_build_args="-DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DBUILD_WITH_MRUBY=OFF -DCMAKE_BUILD_TYPE=Release"
+libretro_tic80_build_args="-DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DBUILD_WITH_MRUBY=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS=-DDINGUX"
 
 include_core_lowresnx() {
 	register_module core "lowresnx" -theos_ios -ngc -sncps3 -ps3 -psp1 -qnx -wii
-- 
2.45.2.windows.1

