From 4fb8e4c2c25aa9a67f65c0b080f06874e95394fe Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Sat, 21 Feb 2026 19:20:20 +0100
Subject: [PATCH] fix picodrive & tic-80 core rules

---
 rules.d/core-rules.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rules.d/core-rules.sh b/rules.d/core-rules.sh
index 9a157fe8..a5aeb026 100644
--- a/rules.d/core-rules.sh
+++ b/rules.d/core-rules.sh
@@ -1138,7 +1138,7 @@ include_core_picodrive() {
 libretro_picodrive_name="Picodrive"
 libretro_picodrive_git_url="https://github.com/libretro/picodrive.git"
 libretro_picodrive_git_submodules="yes"
-libretro_picodrive_build_makefile="Makefile.libretro"
+libretro_picodrive_build_makefile="Makefile.libretro -f ../Makefile.libretro-picodrive"
 
 include_core_tgbdual() {
 	register_module core "tgbdual" -ngc -ps3 -wii
@@ -1504,7 +1504,7 @@ libretro_tic80_git_submodules="yes"
 libretro_tic80_build_subdir="core"
 libretro_tic80_build_extradir="lib/"
 libretro_tic80_build_rule="cmake"
-libretro_tic80_build_args="-DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DBUILD_WITH_MRUBY=OFF -DCMAKE_BUILD_TYPE=Release"
+libretro_tic80_build_args="-DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DBUILD_WITH_MRUBY=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS=-DDINGUX"
 
 include_core_lowresnx() {
 	register_module core "lowresnx" -theos_ios -ngc -sncps3 -ps3 -psp1 -qnx -wii
-- 
2.45.2.windows.1

